sdks/js/keelson/index.ts

```diff
@@ -0,0 +1,50 @@
+import {Envelope} from './Envelope';
```

Comment:
This looks good as far as I can tell! How about helper functions for working with key-expressions? Ref: https://github.com/MO-RISE/keelson/blob/main/sdks/python/keelson/__init__.py#L17-L64